src/plugins/links/public/components/editor/links_editor_link.tsx

```diff
@@ -29,51 +29,47 @@ import {
 import { DashboardContainer } from '@kbn/dashboard-plugin/public/dashboard_container';
 
 import {
-  NavigationLinkType,
+  LinkType,
   EXTERNAL_LINK_TYPE,
   DASHBOARD_LINK_TYPE,
-  NavigationLinkOptions,
-  NavigationEmbeddableLink,
+  LinkOptions,
+  Link,
 } from '../../../common/content_management';
-import { NavigationLinkInfo } from '../../embeddable/types';
-import { NavEmbeddableStrings } from '../navigation_embeddable_strings';
-import { NavigationEmbeddableUnorderedLink } from '../../editor/open_link_editor_flyout';
-import { NavigationEmbeddableLinkOptions } from './navigation_embeddable_link_options';
-import { NavigationEmbeddableLinkDestination } from './navigation_embeddable_link_destination';
+import { LinkInfo } from '../../embeddable/types';
+import { LinksStrings } from '../links_strings';
+import { UnorderedLinks } from '../../editor/open_link_editor_flyout';
+import { LinkOptionsComponent } from './link_options';
+import { LinkDestination } from './link_destination';
 
-export const NavigationEmbeddableLinkEditor = ({
+export const LinksEditorLink = ({
```

Comment:
Some of these component names are so awkward now, oh man 😆  Could we get away with `LinkEditor` here instead?